django/bosscore/serializers.py

```diff
@@ -100,20 +100,28 @@ def validate(self, data):
         # Get the experiment
         exp = data.get('experiment')
         num_time_samples = exp.num_time_samples
-        num_hierarchy_levels = exp.num_hierarchy_levels
+        #num_hierarchy_levels = exp.num_hierarchy_levels
 
         default_time_sample = data.get('default_time_sample', None)
-        base_resolution = data.get('base_resolution', None)
+        #base_resolution = data.get('base_resolution', None)
 
         # Validate that default_time_step is less than the num_time_samples
         if default_time_sample is not None and default_time_sample >= num_time_samples:
             errors['default_time_sample'] = 'Ensure this value is less that the experiments num_time_samples {}.'\
                 .format(num_time_samples)
 
-        # Validate that base_Resolution is less than the num_hierarchy_levels
+        # Validate that base_resolution is less than the num_hierarchy_levels
```

Comment:
I am not following this comment. With the change it is doesn't matter if `base_resolution` is greater than `num_hierarchy_levels`.

Reply:
I was thinking we might want to know that we were originally constraining `base_resolution` to be < `num_hierarchy_levels` but maybe it makes sense to just remove all mention of it.